package.json

```diff
@@ -11,6 +11,7 @@
     "build:clean": "lerna clean -y && lerna bootstrap && lerna run build:clean --concurrency 1 && yarn build:copyfiles && yarn build:schema-api",
     "build:schema-api": "lerna run extract-api && lerna run generate-plugin-schema",
     "build:copyfiles": "lerna run build:copyfiles",
+    "clean": "rimraf --glob **/dist **/coverage packages/**/node_modules node_modules packages/**/tsconfig.tsbuildinfo",
```

Comment:
There already is a build-clean doing roughly the same

Reply:
removed